prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/migrationtoolkit/Notifier.java

```diff
@@ -1,4 +1,4 @@
-package com.redhat.parodos.workflow.task.infrastructure;
+package com.redhat.parodos.tasks.migrationtoolkit;
```

Comment:
perhaps more common package then `migrationtoolkit`? this can be used by others as well.